src/api/asset-management.api.js

```diff
@@ -0,0 +1,5 @@
+import { getHttpRequest } from '@/services/httpService'
+
+export const fetchItemsByCategory = async asset => {
+  return await getHttpRequest(`../../public/api/asset-management/${asset}.json`)
+}
```

Comment:
I like that you extracted it to a separate file. The path is a bit excessive though. See how others did it: https://github.com/iv-thomson/andersen-vue-study/blob/82f2c1f681a6c42c5c4b6fa91f57f0a5c008bb78/src/views/ServiceTicketReportingView.vue#L80